### PR TITLE
Link job board

### PR DESCRIPTION
**Pull request by herzzanu**
Closes https://github.com/ember-learn/ember-website/issues/754

For the record, I'm one of the co-creators of [EMBERWORK](https://emberwork.com/). Thanks for the help @wifelette for this initiative ❤️   and @mansona for your help to work this through 🙏 

#### Home page
image

#### Community page
image

### CODE REVIEW

**Comment by mansona** [comment]
Files that got Bigger 🚨:

File | raw | gzip
--- | --- | ---
ember-website.js|+934 B|+213 B
vendor.js|+803 B|+303 B

Files that stayed the same size 🤷‍:

File | raw | gzip
--- | --- | ---
auto-import-fastboot.js| 0 B| 0 B
ember-website-fastboot.js| 0 B| 0 B
polyfill-evergreen.js| 0 B| 0 B
polyfill-legacy.js| 0 B| 0 B
polyfill-shared.js| 0 B| 0 B
ember-website.css| 0 B| 0 B
vendor.css| 0 B| 0 B

**Comment by herzzanu** [comment]
> Files that got Bigger 🚨:
> 
> File	raw	gzip
> ember-website.js	+934 B	+213 B
> vendor.js	+803 B	+303 B
> Files that stayed the same size 🤷‍:
> 
> File	raw	gzip
> auto-import-fastboot.js	0 B	0 B
> ember-website-fastboot.js	0 B	0 B
> polyfill-evergreen.js	0 B	0 B
> polyfill-legacy.js	0 B	0 B
> polyfill-shared.js	0 B	0 B
> ember-website.css	0 B	0 B
> vendor.css	0 B	0 B

The only reason for this I see is the svg file added. Let me see if I can make that smaller.

**Review by mansona — APPROVED** [review]
I'm happy with this from a design and implementation perspective 🎉  I would wait for a review from @wifelette before we merge though 👍

**Comment by Exelord** [comment]
Awesome! Cannot wait for that ✌️

**Review by Exelord — APPROVED** [review]

File: app/templates/community/index.hbs [hunk 1]
```diff
@@ -20,6 +20,9 @@
     <p>
       <a href="https://dev.to/t/ember" target="_blank" rel="noopener noreferrer">DEV's Ember Community</a> &mdash; a great place to find interesting articles about best practices, application architecture and new learnings. Post your articles with the tag <code>ember</code>.
     </p>
+    <p>
+      <a href="https://emberwork.com/">EMBERWORK</a> &mdash; a dedicated platform for all Ember jobs, carefully curated and validated by our community members.
```
Comment (wifelette) [inline app/templates/community/index.hbs:24]: This is overall good, but we need to be careful about things that sound like official endorsements. Something a bit more generic would be safer. Suggestion:

...a dedicated platform for Ember jobs, curated by dedicated community members.

File: app/templates/index.hbs [hunk 2]
```diff
@@ -239,6 +239,21 @@
         </ul>
       </EsCard>
     </ul>
+
+    <p class="lg:col-4 lg:start-2 my-3 text-center">
+      In search for a job? Our community has your back with a dedicated platform for matching job seekers with the right Ember job opportunities.
```
Comment (wifelette) [inline app/templates/index.hbs:244]: Same line of thinking as my previous comment. Suggestion for this: 

> Looking for your next job? Community members have put together a dedicated platform for matching Ember job seekers with Ember job opportunities.

File: app/templates/index.hbs [hunk 3]
```diff
@@ -239,6 +239,21 @@
...
+    </p>
+
+    <ul class="layout list-unstyled">
+      <EsLinkCard
+        class="lg:col-4 lg:start-2"
+        @icon="emberwork"
+        @title="EMBERWORK - jobs board"
+        @href="https://emberwork.com/"
+      >
+        <p>Find all the Ember job offers in one single place, carefully curated and validated by our community members.</p>
```
Comment (wifelette) [inline app/templates/index.hbs:254]: > Find all the Ember job offers in one single place; a dedicated platform for Ember jobs, curated by community members.

**Review by wifelette — CHANGES_REQUESTED** [review]
Overall looking good! Made a series of copy adjustments, though feel free to edit my words so long as the spirit of the changes is kept 👍🏽

**Comment by herzzanu** [comment]
> Overall looking good! Made a series of copy adjustments, though feel free to edit my words so long as the spirit of the changes is kept 👍🏽

@wifelette Thanks a lot for your feedback and for the copy suggestions. Makes total sense and I 100% agree with it. Just pushed an update to it 🙌

**Review by wifelette — APPROVED** [review]

**Pull request merged**